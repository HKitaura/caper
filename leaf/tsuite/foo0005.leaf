require leaf;

fun main(): int
{
	puti( 'a' );
	puti( 'b' );
	puti( '\0' );
	puti( '\n' );
	puti( '\t' );
	putc( 'a' );
	putc( 'b' );
	putc( '\n' );
	0;
}
