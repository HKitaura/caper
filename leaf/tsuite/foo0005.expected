97
98
0
10
9
ab
